hooks/notification/hook/Notifiers/EMailNotifier.test.ts

```diff
@@ -80,7 +82,40 @@ test("Should Send Mail", async () => {
 
   await notifier.sendMessage();
 
-  expect(sendMail).toBeCalled();
-  expect(close).toBeCalled();
+  expect(sendMail).toBeCalledWith({
+    from: "secureCodeBox",
+    html: `<strong>Scan demo-scan-1601086432</strong><br>
+Created at ${creationTimestamp.toString()}
+<br>
+<br>
+<strong>Findings Severity Overview:</strong><br>
+high: 10<br>
+medium: 5<br>
+low: 2<br>
+informational: 1<br>
+
+<br>
+<strong>Findings Category Overview:</strong><br>
+A Client Error response code was returned by the server: 1<br>
+Information Disclosure - Sensitive Information in URL: 1<br>
+Strict-Transport-Security Header Not Set: 1<br>
+`,
+    subject: "New nmap security scan resulsts are available!",
```

Comment:
Seems like there is a typo somewhere in the code 😁 (proposed edit here just to highlight the typo, the source will be somewhere else).
```suggestion
    subject: "New nmap security scan results are available!",
```

Reply:
Ah haha yes, I thought i fixed it but this was probably in a commits I've later reverted 😅

Reply:
Should be resolved now